docs/content/docs/deployment/advanced/failure_enrichers.md

```diff
@@ -42,7 +42,7 @@ To implement a custom FailureEnricher plugin, you need to:
 
 Then, create a jar which includes your `FailureEnricher`, `FailureEnricherFactory`, `META-INF/services/` and all external dependencies.
 Make a directory in `plugins/` of your Flink distribution with an arbitrary name, e.g. "failure-enrichment", and put the jar into this directory.
-See [Flink Plugin]({% link deployment/filesystems/plugins.md %}) for more details.
+See [Flink Plugin]({{< ref "docs/deployment/filesystems/plugins" >}}) for more details.
```

Comment:
I haven’t reviewed it in detail yet. Can you separate this fix change into an independent commit? This is unrelated to translation work.

Reply:
Good advice, here I fixed it